src/Component/Style/Style.tsx

```diff
@@ -190,6 +191,35 @@ export class Style extends React.Component<StyleProps, StyleState> {
     this.setState({style});
   }
 
+  cloneRules = () => {
+    const {
+      selectedRowKeys,
+      style
+    } = this.state;
+    const styleClone = _cloneDeep(style);
+
+    // create rules to clone
+    let newRules: GsRule[] = [];
+    styleClone.rules.forEach((rule: GsRule, index: number) => {
+      if (selectedRowKeys.includes(index)) {
+        let ruleClone = _cloneDeep(rule);
+        // TODO We need to ensure that rule names are unique
+        const randomId = Math.floor(Math.random() * 10000);
```

Comment:
Huh ;-) Maybe use https://github.com/kelektiv/node-uuid for this?

Reply:
Agreed we need to fix this. But in a separate PR, because we had this discussion elsewhere? Do you agree?

Reply:
Maybe lodash's [uniqueId()](https://lodash.com/docs/#uniqueId) can be helpful here. We also already use it in this project, as far as I know.